state-chain/pallets/cf-ingress-egress/src/tests.rs

```diff
@@ -439,6 +440,7 @@ fn reused_address_channel_id_matches() {
 		let (reused_channel_id, reused_address, ..) = IngressEgress::open_channel(
 			eth::Asset::Eth,
 			ChannelAction::LiquidityProvision { lp_account: 0 },
+			0,
```

Comment:
We could add a simple test to check that when the fee is passed in that this same fee is emitted in the event - might make more sense to add this in the caller tests (so the swapping pallet or LP pallet) though